### DISCUSSION TITLE
How do I set up dolphin-mixtral:8x7b as a model I can use in elia?

### DISCUSSION THREAD

**Discussion by mearleycf** [darrenburns/elia#58] [Questions]
[Dolphin Mixtral via Ollama](https://ollama.com/library/dolphin-mixtral:8x7b)
I know I'd do it using a [[models]] table. Is it as simple as:
```
[[models]]
name = "ollama/dolphin-mixtral:8x7b"
```

**Comment by darrenburns** [comment]
I believe so!

(assuming you've pulled the model via ollama as mentioned in the readme)
